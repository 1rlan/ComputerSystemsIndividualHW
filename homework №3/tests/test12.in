-7983146791380467.3489671398467938
