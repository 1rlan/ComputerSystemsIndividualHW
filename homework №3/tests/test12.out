-199859.457857
